src/sentry/constants.py

```diff
@@ -86,7 +86,7 @@ def get_all_languages():
         'pricing', 'subscribe', 'enterprise', 'about', 'jobs', 'thanks', 'guide', 'privacy',
         'security', 'terms', 'from', 'sponsorship', 'for', 'at', 'platforms', 'branding', 'vs',
         'answers', '_admin', 'support', 'contact', 'onboarding', 'ext', 'extension', 'extensions',
-        'plugins', 'themonitor', 'settings',
+        'plugins', 'themonitor', 'settings', '_',
```

Comment:
Do we actually need to reserve this? I thought we had a min length of like, 3 characters anyways for an org name.